eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -1,15 +1,15 @@
 # All required SDKs/runtimes are available on DevOps agents
-steps: []
+#steps: []
```

Comment:
Nit: any reason to keep this? Or are these two alternative sections just meant to be flipped when an agent is behind?

Reply:
Yeah, we are flipping them when we need an update.